tox.ini

```diff
@@ -24,6 +24,7 @@ deps =
     -r{toxinidir}/requirements/anomaly.txt
     -r{toxinidir}/requirements/openvino.txt
 commands = 
+    pip install -e .[classification]
```

Comment:
how about `pip install -e .[full]` instead of `pip install -e .[classification] & pip install -e .[detection]`?
could you check this?

Reply:
Isn't [full] install anomaly too? The previous code base did not install anomaly. Could you answer why it was?

Reply:
Existing installations also contain anomaly. does not matter.